okapi-common/src/main/java/org/folio/okapi/common/SemVer.java

```diff
@@ -74,14 +95,27 @@ private int parseComp(String v, boolean digits, int offset, List<String> result)
     return i;
   }
 
+  /**
+   * Whether the version appears to be an NPM snapshot (at least in FOLIO)
+   * @return true if it appears to be an NPM snapshot; false otherwise
+   */
   public boolean hasNpmSnapshot() {
     return versions.size() == 3 && versions.get(2).length() >= 5;
   }
 
+  /**
+   * Whether version has pre-release
+   * @return true if version has pre-release; false otherwise
+   */
   public boolean hasPreRelease() {
     return !preRelease.isEmpty();
   }
 
+  /**
+   * Whether this version has prefix of other
```

Comment:
Can you explain in the Javadoc what a prefix is in the context of semantic versioning? I cannot find it on https://semver.org/spec/v1.0.0.html

Reply:
See comments for previous review item.. I typed in the wrong box